build.gradle

```diff
@@ -118,6 +118,11 @@ subprojects { subproject ->
     }
   }
 
+  tasks.withType(JavaCompile).configureEach {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
```

Comment:
This is a breaking change that we already did in the past and had to revert. Is now a good time to pull the trigger? I don't see why not but wanted to double-check @mapbox/navigation-android @mapbox/maps-android. The next mapbox-java release will be a major one anyway.

We'll also need to update https://docs.mapbox.com/android/java/guides/install/, could you cut a ticket @korshaknn if this is merged?

Reply:
can't bump okhttp to 4.x without  this `javaVersion` changes